src/electron/window.main.ts

```diff
@@ -118,6 +118,7 @@ export class WindowMain {
             alwaysOnTop: this.enableAlwaysOnTop,
             webPreferences: {
                 nodeIntegration: true,
+                nodeIntegrationInWorker: true,
```

Comment:
is this still necessary? looks like you stopped using node in the worker.

Reply:
Good point - I've now removed this and cleaned up some node dependencies that were accidentally being used in the worker. I think the new boolean logic in `utils.ts` (`isWebWorker` etc) will need to be looked at closely. But this is great because we avoid Node threadsafety issues, afaik.